mmrazor/models/quantizers/native_quantizer.py

```diff
@@ -96,16 +134,30 @@ def prepare(self, model, graph_module):
 
     def post_process_weight_fakequant(self,
                                       observed_module,
-                                      keep_fake_quant=False):
+                                      keep_fake_quant: bool = False):
+        """weight fakequant for supported QAT modules.
+
+        Args:
+            observed_module (_type_): _description_
+            keep_fake_quant (bool, optional): _description_. Defaults to False.
+        """
 
         def traverse(module):
             for name, child in module.named_children():
+                # Trace `SUPPORT_QAT_MODULES` recursively.
                 if isinstance(child, SUPPORT_QAT_MODULES):
+                    # We add w_fakequant once in case some ptq methods have
+                    # specific operations such as Adaround. So we do Quantize
+                    # to perform these operations and do dequantize to
+                    # introduce quantization loss in advance.
                     weight_fakequant = child.weight_fake_quant
                     child.weight.data = weight_fakequant(child.weight.data)
 
+                    # `to_float()` function fuse BN into conv or conv_relu.
```

Comment:
`to_float()` is not only fuse BN but also will convert a qat module to a normal module.

Add `to_float()` source code url